app/views/govuk_publishing_components/components/_layout_for_admin.html.erb

```diff
@@ -25,5 +25,8 @@
     <% end -%>
     <%= yield %>
     <%= javascript_include_tag js_filename %>
+    <% if GovukPublishingComponents::Config.use_es6_components %>
+      <%= javascript_include_tag 'es6-components', type: "module" %>
+    <% end %>
```

Comment:
This has a hardcoded filename compared the configurable ones. 

Would we be able to have a js_module_filename variable? 

Also, should this actually be in the head element given it's loading is deferred?

Reply:
Thanks for the feedback Kevin, I've made the changes as suggested.

I've moved the script to the the head section and added the `js_module_filename` variable in the admin layout template.